src/core/oned/Code128Reader.ts

```diff
@@ -199,7 +199,7 @@ export default class Code128Reader extends OneDReader {
           }
           patternStart += counters[0] + counters[1];
 
-          counters = counters.slice(2, counters.length - 1);
+          counters = counters.slice(2, counters.length);
```

Comment:
Unfortunately splice() cannot be used here, as that method is only available on normal arrays, but `counters` is a TypedArray. 

The code indeed used to use splice() prior to 0.18.3 when `counters` was just a regular array.

And as to why I chose to go with the longer `counters = counters.slice(2, counters.length);` - it's because that's what #536 did and I figured I'd just do the same for the sake of consistency, but you are right, the second argument is unnecessary.

Reply:
@JanPolasek Good catch!